src/screen/DemocracyProposalScreen.tsx

```diff
@@ -0,0 +1,235 @@
+import {RouteProp} from '@react-navigation/native';
+import {Button, Card, Divider, Icon, Layout, Modal, Text} from '@ui-kitten/components';
+import {useApi} from 'context/ChainApiContext';
+import {useTX} from 'context/TxContext';
+import AddressInlineTeaser from 'layout/AddressInlineTeaser';
+import {EmptyView} from 'presentational/EmptyView';
+import LoadingView from 'presentational/LoadingView';
+import Padder from 'presentational/Padder';
+import {ProposalInfo} from 'presentational/ProposalInfo';
+import SafeView, {noTopEdges} from 'presentational/SafeView';
+import {SelectAccount} from 'presentational/SelectAccount';
+import React, {useReducer, useState} from 'react';
+import {StyleSheet, TouchableOpacity, View} from 'react-native';
+import {ScrollView} from 'react-native-gesture-handler';
+import {useDemocracy} from 'src/api/hooks/useDemocracy';
+import {useFormatBalance} from 'src/api/hooks/useFormatBalance';
+import {DashboardStackParamList} from 'src/navigation/navigation';
+import {referendumScreen} from 'src/navigation/routeKeys';
+import globalStyles, {standardPadding} from 'src/styles';
+
+export function DemocracyProposalScreen({route}: {route: RouteProp<DashboardStackParamList, typeof referendumScreen>}) {
+  const {start} = useTX();
+  const {api} = useApi();
+
+  const [secondsOpen, setSecondsOpen] = useState(false);
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  const formatBalance = useFormatBalance();
+  const {data, isLoading} = useDemocracy();
+  if (isLoading) {
+    return <LoadingView />;
+  }
+  const activeProposal = data?.activeProposals.find((p) => String(p.index) === route.params.index);
+  if (!activeProposal) {
+    return <EmptyView />;
+  }
+
+  const proposal = activeProposal.image?.proposal;
+  const {method, section} = proposal?.registry.findMetaCall(proposal.callIndex) ?? {};
+
+  return (
+    <Layout style={globalStyles.flex}>
+      <SafeView edges={noTopEdges}>
+        <ScrollView style={styles.container}>
+          <Text category={'h5'}>Proposal</Text>
```

Comment:
Let's remove this and set it as title in the screen's header.

Reply:
done ✅<img align="right" src=https://user-images.githubusercontent.com/1168983/128718622-2d796a4e-5bcb-453b-8322-c95252329ee0.png width=200 />